src/resources/postcss/tribe-common-admin/settings/_page.pcss

```diff
@@ -6,66 +6,79 @@
  *
  * ----------------------------------------------------------------------------- */
 
-.tribe_events_page_tec-events-settings {
-	#wpcontent {
-		padding-left: 0;
-	}
-
-	#wpbody-content {
-		@media (max-width: 782px) {
-			padding-bottom: 0;
-		}
-	}
-
-	#tec-settings-logo {
-		margin: var(--tec-spacer-2);
-		margin-left: var(--tec-spacer-5);
-		vertical-align: middle;
-		width: var(--tec-spacer-7);
-
-		@media (max-width: 782px) {
-			margin: var(--tec-spacer-2);
-		}
-	}
-
-	.tribe_settings.wrap {
-		display: flex;
-		flex-flow: column nowrap;
-		margin: 0;
-
-		@media (--viewport-xlarge) {
-			flex-flow: row wrap;
-			justify-content: flex-start;
-			padding-bottom: var(--tec-spacer-7);
-		}
-
-		.tribe-notice-wrap {
-			background-color: var(--tec-color-background);
-			width: 100%;
-
-			> div {
-				border-radius: var(--tec-border-radius-default);
-				box-sizing: border-box;
-				margin: var(--tec-spacer-1) var(--tec-spacer-7);
-
-				@media (min-width: 500px) and (max-width: 1259px) {
-					margin: var(--tec-spacer-1) var(--tec-spacer-5);
-				}
-			}
-		}
-	}
-
-	.tribe_settings.wrap h1 {
-		background-color: var(--tec-color-background);
-		font-family: var(--tec-font-family-sans-serif);
-		font-size: var(--tec-font-size-7);
-		font-weight: var(--tec-font-weight-regular);
-		max-width: 100%;
-		padding: var(--tec-spacer-2) var(--tec-spacer-5);
-		width: auto;
-
-		@media (--viewport-xlarge) {
-			width: 100%;
-		}
-	}
+ .tribe_events_page_tec-events-settings {
+  #wpcontent {
```

Comment:
Should this have an indent here?

Reply:
The linter had formatted it like this, don't know why. Added one indentation, thanks!